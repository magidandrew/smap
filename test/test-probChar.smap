int main(){
	prob char c = [0.3,0.7,0.25] : ['a','b','v'];
	prob char c2 = c;
	int i;
	list float probs = c#;
	for(i=0; i < c.length; i = i+1){
		println(probs[i]);
	}
    //print(c!);
}