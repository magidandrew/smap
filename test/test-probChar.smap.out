0.240000
0.559990
0.200010
